app/models/miq_server/worker_management/monitor/kubernetes.rb

```diff
@@ -15,33 +20,88 @@ def failed_deployments(restart_count = 5)
     current_pods.values.select { |h| h[:last_state_terminated] && h.fetch(:container_restarts, 0) > restart_count }.collect { |h| h[:label_name] }
   end
 
+  def sync_deployment_settings
+    return unless podified?
+    return unless Settings.server.worker_monitor.enforce_resource_constraints
```

Comment:
Does this imply that the only thing that we sync are resource constraints? I wonder if it's better to move this into the `deployment_resource_constraints_changed?` method instead.  That way if we need to patch for other reasons it's not muddied by this guard.

Reply:
Fixed, good 👁️